src/components/ViewRequest/HiringPanel.tsx

```diff
@@ -0,0 +1,64 @@
+import { Label, Switch, Text } from "@fluentui/react-components";
+import { usePostRequest } from "api/postRequestApi";
+import { useRequest } from "api/requestsApi";
+import { STAGES } from "consts/Stages";
+import { ChangeEvent, useCallback } from "react";
+import { useParams } from "react-router-dom";
+
+const HiringPanel = () => {
+  const params = useParams();
+  const request = useRequest(Number(params.requestId));
+  const postRequest = usePostRequest();
+
+  const onChange = useCallback((ev: ChangeEvent<HTMLInputElement>) => {
+    postRequest.mutate({
+      requestId: Number(params.requestId),
+      postRequest: { panelRequired: ev.currentTarget.checked ? "Yes" : "No" },
+    });
+  }, []);
+
+  const selectionStageIndex = STAGES.findIndex(
+    (stage) => stage.key === "Selection"
+  );
+  const currentStageIndex = STAGES.findIndex(
+    (stage) => stage.key === request.data?.stage
+  );
+
+  if (currentStageIndex < selectionStageIndex) {
+    return <></>;
+  }
+
+  if (currentStageIndex > selectionStageIndex) {
```

Comment:
How do you want to handle Cancelled status?  Since we change the stage itself to Cancelled we can't determine the state it was in when cancelled, however, we could just say if it has a value then show, and if it has no value then hide?